app/src/main/java/ru/androidacademy/gooseeye/ArtistRecyclerAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package ru.androidacademy.gooseeye
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.imageview.ShapeableImageView
+
+class ArtistRecyclerAdapter(private val values: List<ArtistInfo>) :
+    RecyclerView.Adapter<ArtistRecyclerAdapter.ArtistViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ArtistViewHolder {
+        val itemView = LayoutInflater.from(parent.context)
+            .inflate(R.layout.item_artist, parent, false)
+        return ArtistViewHolder(itemView)
+    }
+
+    override fun onBindViewHolder(holder: ArtistViewHolder, position: Int) {
+        holder.artistPhoto?.setImageResource(values[position].artistPhoto)
+        holder.artistName?.text = values[position].artistName
```

Comment:
it is good practice to add `fun bind(artistInfo: ArtistInfo)` to holder class and fill it there